site/gatsby-site/src/pages/unsubscribe.js

```diff
@@ -62,8 +62,10 @@ const Unsubscribe = (props) => {
       await DeleteSubscriptions({ variables: { query } });
 
       setPageMessage(t('You have successfully unsubscribed.'));
-    } catch (error) {
+    } catch (e) {
       setPageMessage(t('An unknown error has ocurred'));
```

Comment:
This is the only page showing a message on the page instead of a toast. A  bit of inconsistency, but for another issue.

Reply:
issue https://github.com/responsible-ai-collaborative/aiid/issues/1675 created